pkg/gce-cloud-provider/compute/cloud-disk.go

```diff
@@ -223,3 +223,14 @@ func (d *CloudDisk) GetMultiWriter() bool {
 		return false
 	}
 }
+
+func (d *CloudDisk) GetEnableConfidentialStorage() bool {
+	switch {
+	case d.disk != nil:
+		return false
+	case d.betaDisk != nil:
+		return d.betaDisk.EnableConfidentialCompute
```

Comment:
Side note, I tried to search the string EnableConfidentialCompute in this PR but it only showed up here. You need to pass in EnableConfidentialCompute to the Disk struct for disk.insert GCP PD APIs so I guess that is missing. The disk.insert will not recognize EnableConfidentialStorage.

Reply:
Post our discussion, we decided to rename all values to `EnableConfidentialCompute` to match with PD team